test/cli.test.ts

```diff
@@ -37,6 +37,32 @@ export default defineConfig({
     expect(output).toMatchSnapshot()
   })
 
+  it('supports variantGroup transformer', async () => {
+    const { output } = await runCli({
+      'views/index.html': '<div class="p-4 border-(solid red)"></div>',
+      'unocss.config.js': `
+import { defineConfig, transformerVariantGroup } from 'unocss'
+export default defineConfig({
+  transformers: [transformerVariantGroup()]
+})
+    `.trim(),
+    })
+    expect(output).toMatchSnapshot()
+  })
+
+  it('supports directives transformer', async () => {
+    const { output } = await runCli({
```

Comment:
`runCli` only output `uno.css` file content. I would prefer to see the tranformed code.